### DISCUSSION TITLE
Listo mi Pokedex

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#447] [Show and tell]
Ya terminé de afinar unos detalles de mi Pokedex, díganme qué les parece?

https://pokedex.animeplay.us

En movil no me agradó mucho pero creo que quedó bien

**Comment by AlfredoLandero** [comment]
Disculpa como es que se ponen imágenes de mejor calidad?. Las que vienen en el api del ejemplo están algo pixeladas.

**Reply by marco-cortes** [reply]
En la información de la api hay varias imágenes, las que usa están en
sprites.other.home.front_default

**Reply by sergiolopezloya** [reply]
Precisamente esa ruta te iba a comentar, yo lo tengo así mismo: pokemon.sprites.other.home.front_default
